K2Bridge/Factories/BucketFactory.cs

```diff
@@ -23,12 +24,12 @@ public static DateHistogramBucket CreateDateHistogramBucketFromDataRow(DataRow r
         {
             Ensure.IsNotNull(row, nameof(row));
 
-            // TODO: timestamp is always the first row (probably named _2), and count will be named _count
+            // TODO: timestamp is always the first column (probably named 2), and count will be named _count
```

Comment:
```suggestion
            // TODO: The 'summary by' field is always the first column and count will be named _count
```

Reply:
I have reverted to `timestamp`.

Reply:
here, this is the place where the project-reorder might cause issues